app/src/main/res/layout/beer_layout.xml

```diff
@@ -36,9 +35,9 @@
             <TextView
                 android:id="@+id/title"
                 style="@style/beer_name"
-                android:layout_width="130dp"
```

Comment:
è meglio lasciare "wrap_content" in quanto se il titolo della birra è troppo lungo non entra nella card